projects/kit/src/lib/masks/number/utils/parse-number.ts

```diff
@@ -1,14 +1,22 @@
 import {CHAR_EM_DASH, CHAR_EN_DASH, CHAR_HYPHEN, CHAR_MINUS} from '../../../constants';
 import {escapeRegExp} from '../../../utils';
+import {extractPrefixAndPostfix} from './extract-prefix-and-postfix';
 
 export function maskitoParseNumber(
```

Comment:
I've update utility `maskitoParseNumber` in this PR https://github.com/taiga-family/maskito/pull/874
I think we could revert changes for this file.

As for **postfix with decimal separator**, `maskitoParseNumber` can parse it now without additional argument:
```ts
maskitoParseNumber('1 000.42 lbs.') // 1 000.42
```

___

As for **prefix with decimal separator**, `maskitoParseNumber` can parse it now without additional argument too:
```ts
maskitoParseNumber('lbs. 1 000.42') // 1 000.42
```

The **only exception** – when prefix is ended with decimal separator and does not separated from number by space or any other characters.
```ts
maskitoParseNumber('lbs. .42') // 0.42
maskitoParseNumber('lbs.42') // 42 or 0.42 ?
```

I think it very rare case and we can add simple workaround for it.
I will explain everything in separate comment.

Reply:
I think for this rare case we can automatically pad prefix with zero-width space.

**projects/kit/src/lib/constants/unicode-characters.ts:**
```ts
export const CHAR_ZERO_WIDTH_SPACE = '\u200B';
```


**projects/kit/src/lib/masks/number/number-mask.ts:**
```ts
export function maskitoNumberOptionsGenerator({
    decimalSeparator = '.',
    prefix = '',
    // [...]
}): Required<MaskitoOptions> {
    const safePrefix = prefix.endsWith(decimalSeparator)
        ? `${prefix}${CHAR_ZERO_WIDTH_SPACE}`
        : prefix;
    // [...]
}
```

Reply:
I will create the separate PR to implement zero-width workaround